AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/assign-quotas-dialog/assign-quotas-dialog.component.html

```diff
@@ -31,6 +31,9 @@ <h2 mat-dialog-title>{{ header }}</h2>
         <mat-progress-spinner color="primary" mode="indeterminate"> </mat-progress-spinner>
     </div>
 </mat-dialog-content>
+<div *ngIf="errorMessage !== '' && loading" class="quota-error-container">
```

Comment:
Why does it only show if data is currently loading?

Reply:
My mistake, i overlooked that one.